### PR TITLE
MapAnnotation Pojo and Tests 

### PR DESCRIPTION
**Pull request by dominikl**
Adds the Pojo for MapAnnotations and the integration tests.
Test: Check that the (annotation related) integration tests pass

(This PR also adds a namespace constant for client created MapAnnotations to Constants.ice /cc @will-moore  )

--no-rebase
--depends-on #3334

### CODE REVIEW

**Comment by jburel** [comment]
Discussed with @dominikl, removing the exclude label so we can test the UI work.

**Comment by dominikl** [comment]
--depends-on #3325

**Comment by mtbc** [comment]
Appears to break a bunch of integration tests, e.g., in DeleteServiceTest.

File: components/blitz/resources/omero/Constants.ice [hunk 1]
```diff
@@ -158,6 +158,7 @@ module omero {
         const string NSINSIGHTTAGSET = "openmicroscopy.org/omero/insight/tagset";
         const string NSINSIGHTRATING = "openmicroscopy.org/omero/insight/rating";
         const string NSMOVIE = "openmicroscopy.org/omero/movie";
+        const string NSCLIENTMAPANNOTATION = "openmicroscopy.org/omero/client/mapAnnotation";
```
Comment (joshmoore) [inline components/blitz/resources/omero/Constants.ice]: I'm not quite sure about the use of this namespace. How does this differ from just any map annotation?

Comment (jburel) [inline components/blitz/resources/omero/Constants.ice]: The Idea was to have one map annotation that the user will be editing. One per user.
Other could be generated during acquisition are not editable in clients.
This was discussed in MapAnnotation mockup

**Comment by dominikl** [comment]
Looks like MapAnnotations can't be deleted or moved. I assume it's not an issue of the tests, but the delete and move methods have to be adjusted to handle MapAnnotations. Can someone give me a hint where to do this (I'm still not really familiar with the server side code)?

File: components/tools/OmeroJava/test/integration/AbstractServerTest.java [hunk 2]
```diff
@@ -1604,10 +1615,20 @@ public void run(omero.util.TileData data, int z, int c, int t,
...
+		ma.setMapValue(values);
+		if (ns != null)
+			b.setNs(omero.rtypes.rstring(ns));
+		ma = (MapAnnotation) iUpdate.saveAndReturnObject(ma);
+        
         ids.add(c.getId().getValue());
         ids.add(l.getId().getValue());
         ids.add(b.getId().getValue());
-
+        ids.add(ma.getId().getValue());
```
Comment (mtbc) [inline components/tools/OmeroJava/test/integration/AbstractServerTest.java]: also need to add `ma` into code below to actually create the link?

**Comment by dominikl** [comment]
Travis is continuously failing, without a sensible error message (as far as I can see); don't know why.

File: components/tools/OmeroJava/test/integration/DeleteServiceTest.java [hunk 3]
```diff
@@ -201,7 +201,10 @@
 
     /** Identifies the File. */
     public static final String REF_FILE = "/FileAnnotation";
-
+    
+    /** Identifies the File. */
```
Comment (mtbc) [inline components/tools/OmeroJava/test/integration/DeleteServiceTest.java]: the file?

**Comment by dominikl** [comment]
Fixed the wrong comment. If it's ok otherwise, would be great if it could be merged soon, so I can open my Insight UI PR for the MapAnnotations to look at.

**Comment by joshmoore** [comment]
:+1: for the comment fixed. Any discussion on the namespace can take place elsewhere. Merging.

**Pull request merged**